crypto/keys/internal/ecdsa/pubkey.go

```diff
@@ -46,9 +56,26 @@ func (pk *PubKey) Bytes() []byte {
 }
 
 // VerifySignature checks if sig is a valid ECDSA signature for msg.
+// This includes checking for low-s normalized signatures
+// where the s integer component of the signature is in the
+// lower half of the curve order
+// 7/21/21 - expects raw encoded signature (fixed-width 64-bytes, R || S)
+
 func (pk *PubKey) VerifySignature(msg []byte, sig []byte) bool {
-	s := new(signature)
-	if _, err := asn1.Unmarshal(sig, s); err != nil || s == nil {
+
+	// check length for raw signature
+	// which is two 32-byte padded big.Ints
+	// concatenated
+	// NOT DER!
+
+	if len(sig) != 64 {
+		return false
+	}
+
+	s := signatureFromBytes(sig)
+
+	// if not low-s, then fail validation
+	if !IsSNormalized(s.S) {
```

Comment:
```suggestion
	s := signatureFromBytes(sig)
	if !IsSNormalized(s.S) {
```

Reply:
The code is self explanatory: _if not normalized then false_ (no need for additional comment).